tests/shell/check_meta.sh

```diff
@@ -0,0 +1,39 @@
+@INCLUDE_COMMON@
+
+echo
+echo ELEKTRA PLUGIN METATADA CONSISTENCY CHECK
+echo
+
+PLUGINS_DIR="@CMAKE_SOURCE_DIR@/src/plugins"
+META_FILE="@CMAKE_SOURCE_DIR@/doc/METADATA.ini"
+
+for PLUGIN in `ls "$PLUGINS_DIR"`;
+do
+
+    if [ ! -d "${PLUGINS_DIR}/${PLUGIN}" ];
+    then
+	continue
+    fi
+    
+    README="${PLUGINS_DIR}/${PLUGIN}/README.md"
+    PLUGIN_META=$(grep -Eo "infos/metadata(.*)" "$README" |cut -d '=' -f2)
+    for META in $PLUGIN_META;
+    do
+	grep -Eq "^\\[${META}\\]$" "$META_FILE"
+        succeed_if "Metadata $META of plugin $PLUGIN not present in METADATA.ini"
+    done
+    USED_BY=$(awk "/usedby\\/plugin= ([^\\n]*)${PLUGIN}([^\\n]*)/" RS= "$META_FILE")
```

Comment:
@sanssecours @KurtMi 
any idea how to fix the awk command to not match "condition/calculate/etc" for "c"